pkg/controllers/cspc-controller/sync.go

```diff
@@ -438,3 +458,17 @@ func (pc *PoolConfig) patchPoolDeploymentSpec(cspi *cstor.CStorPoolInstance) err
 
 	return nil
 }
+
+func (c *Controller) ShouldReconcile(cspc cstor.CStorPoolCluster) (bool, string) {
+	cspcOperatorVersion := version.Current()
```

Comment:
We will maintain Version file in cstor-opeeraters repo instead of depending on Maya

Reply:
Will migrate version package to cstor-operators

Reply:
@mittachaitu -- migration of version package will be taken in a separate PR.